### DISCUSSION TITLE
Using add-data to add a folder, and the folder isn't in the temp folder that the EXE is running from

### DISCUSSION THREAD

**Discussion by cyberducks17** [pyinstaller/pyinstaller#8877] [Help]
I used 
--add-data .\scripts;scripts
and it worked
but when I ran the python code and checked the temp folder
the folder is not there

**Comment by rokm** [comment]
Works as expected for me, so you'll have to provide more details/context - for example, is this onedir or onefile build, and where did you expect the directory to be?
